action.yaml

```diff
@@ -158,6 +158,22 @@ runs:
         JF_GIT_TOKEN: ${{ env.JF_GIT_TOKEN }}
         JF_WORKING_DIR: ${{ env.JF_WORKING_DIR }}
 
+    - name: Creating Kind Cluster 
+      id: cluster-creation
+      if: ${{env.KIND_CLUSTER_NAME }}
```

Comment:
Couldn't this be
`if: ${{env.KIND_CLUSTER_NAME && env.KIND_CLUSTER_ACTION == 'CREATE' }}`

and when deleting:
`if: ${{env.KIND_CLUSTER_NAME && env.KIND_CLUSTER_ACTION == 'DELETE' }}`
?

What do you think?

Reply:
yes we can go with this one